pam/integration-tests/testdata/TestCLIIntegration/golden/exit_authd_if_user_sigints

```diff
@@ -132,8 +132,10 @@ Gimme your password
 ────────────────────────────────────────────────────────────────────────────────
 > ./pam_authd socket=/tmp/pam-cli-tests.sock
 Gimme your password
-Auth return: Critical error - immediate abort: cancel requested
-Acct mgmt return: The return value should be ignored by PAM dispatch
+PAM ERROR: cancel requested
```

Comment:
ditto here. Please review all error messages to ensure we don’t stutter the error to the user

Reply:
As said in https://github.com/ubuntu/authd/pull/155#discussion_r1455250585 and https://github.com/ubuntu/authd/pull/155#discussion_r1455647707 the PAM error messages are shown as last call, and they do need to be sent.

Then it's up to the UI in use to show them. In the interactive mode those are not shown for real, but we do manage them inside the test client in order to test what we're actually doing.

Reply:
So I've changed this so that this specific error will be instead:

```
PAM Error Message: cancel requested
PAM Authenticate() for user 'user-integration-sigint' exited with error (PAM exit code: 26): Cri
tical error - immediate abort: cancel requested
PAM AcctMgmt() exited with error (PAM exit code: 25): The return value should be ignored by PAM
dispatch
```

I guess it's gonna be clear that messages are the ones printed to the user, the rest is just an error code that is sent back to `pam_sm_authenticate()`.